### PR TITLE
桌上應該要有菜色

### PR DESCRIPTION
**Pull request by Dragonfly911117**
所以我去野外用針抓了一隻兔子回來做菜

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,5 @@
 - Chair: 6
 
 - Table: 2
+
+- 雜燴兔:1016
```
Comment (ntut-xuan) [inline README.md:7]: 我認為應該要有三色豆

**Review by ntut-xuan — CHANGES_REQUESTED** [review]
菜色再新增一道。

File: README.md (hunk 1 above)
Comment (Dragonfly911117) [reply README.md:7]: No!

**Review by kyynk — APPROVED** [review]

**Review by SunnyChan0526 — APPROVED** [review]

**Review by OuChiaYun — APPROVED** [review]

**Review by garry911128 — APPROVED** [review]

**Comment by Dragonfly911117** [comment]
Since we don't think the dishes is needed, I'm closing the PR.

**Pull request closed without merging**